content/posts/cneuromonth_videos.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "CNeuromonth videos"
+description: "The videos of our presentations for the CNeuromont 2021 are available on our youtube channel."
+date: 2021-04-21 
+author: "François Paugam"
+image: "cneuromonth_videos/youtube-logo.png"
+draft: false
+tags: ["cneuromonth"]
+---
+
+Hello, I am François Paugam, a PhD student and the one in chatrge of the special effects for the great videos we made for the CNeuromod month, our online progress report events. If you missed it, or if you are just curious to see what we are doing in the numerous and diverse branches of the Neuromod project, or if you enjoyed the presentations so much that you want to watch them over and over, you can access them on our [youtube channel](https://www.youtube.com/channel/UCjeuWWEmZt5Zbx0LBiPbDJw).
```

Comment:
minor comment: perhaps better to stick with CNeuromod everywhere?

Reply:
Yes that would be better, I also changed CNeuromonth for CNeuromod month to be consistant with Julie's previous post.